src/vs/workbench/contrib/notebook/browser/notebookEditorKernelManager.ts

```diff
@@ -25,7 +25,7 @@ export class NotebookEditorKernelManager extends Disposable {
 	getSelectedOrSuggestedKernel(notebook: INotebookTextModel): INotebookKernel | undefined {
 		// returns SELECTED or the ONLY available kernel
```

Comment:
Minor, but this comment looks like it should be updated along with this change. It returns selected or the suggested kernel if there is only one suggestion.

Reply:
Oh yes, that's still a bug then, thantks

Reply:
I'm not sure if that update is correct. I was thinking that your code was right before but the comment in the function was wrong. Is that not the case? I was reading this fix as:

Selected if available. If not, suggested if only one suggested.

Which is what you had before. What it's changed to now you would return undefined if there was only one suggested and 2+ total kernels.

Reply:
Agreed, will do that.
Looks like I'll need to submit this from another machine (my wsl where my commits are signed & SSH setup properly)

Reply:
We cannot, 
* If user has only one kernel, we should use that withouth prompting
* If user has a selected kernel, we should use that without prompting
* If user doesn't have any kiernels, do nothing
* In all other cases we should prompt the user to select a kernel